dot/sync/chain_sync.go

```diff
@@ -587,21 +640,19 @@ func (cs *chainSync) requestMaxBlocksFrom(bestBlockHeader *types.Header, origin
 // head block number, it would leave us in bootstrap mode forever
 // it would be better to have some sort of standard deviation calculation and discard any outliers (#1861)
```

Comment:
I think this was solved https://github.com/ChainSafe/gossamer/pull/1992 so we can use this PR to remove this comment too 😄